linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_retry/tests/correct_value.pass.sh

```diff
@@ -1,8 +1,12 @@
 #!/bin/bash
-# platform = Red Hat Enterprise Linux 7,Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_wrlinux
```

Comment:
This change causes that this scenario will be executed also on RHEL8, but it shouldn't be executed on RHEL8 because it fails on RHEL8 and there is a different scenario only for RHEL8 rhel8_correct_value.pass.sh. You will have to list Ubuntu here instead of removing the line.

Reply:
fixed, thanks!